### ISSUE TITLE
Error accessing file, this might be due to fastlane's directory handling

### ISSUE THREAD

**Issue by lakamsani** [fastlane/fastlane#10642]
### Issue Description
Started getting this error on one of our dev laptops. Same behavior with FL 2.62.0. 

### Captured Output

Command Used: `jenkins app:dwts`
<details><summary>Output/Log</summary>

```

DEBUG [2017-10-19 00:20:23.08]: Checking if there are any plugins that should be loaded...
DEBUG [2017-10-19 00:20:23.08]: Loading 'fastlane-plugin-upload_symbols_to_new_relic' plugin
INFO [2017-10-19 00:20:23.11]: gem 'fastlane-plugin-upload_symbols_to_new_relic' is already installed
DEBUG [2017-10-19 00:20:23.12]: Loading 'fastlane-plugin-appicon' plugin
INFO [2017-10-19 00:20:23.15]: gem 'fastlane-plugin-appicon' is already installed
+------------------------+---------+------------------------+
|                       Used plugins                        |
+------------------------+---------+------------------------+
| Plugin                 | Version | Action                 |
+------------------------+---------+------------------------+
| fastlane-plugin-uploa  | 0.1.1   | upload_symbols_to_new  |
| d_symbols_to_new_reli  |         | _relic                 |
| c                      |         |                        |
| fastlane-plugin-appic  | 0.10.0  | android_appicon        |
| on                     |         | appicon                |
+------------------------+---------+------------------------+



WARN [2017-10-19 00:20:23.28]: Error accessing file, this might be due to fastlane's directory handling
WARN [2017-10-19 00:20:23.28]: Check out https://docs.fastlane.tools/advanced/#directory-behavior for more details

bundler: failed to load command: fastlane (/Users/vamsee/Development/ruby/bin/fastlane)
Errno::ENOENT: No such file or directory - SW_VERS
  /Users/vamsee/Development/ruby/gems/fastlane-2.62.0/fastlane_core/lib/fastlane_core/analytics/analytics_session.rb:194:in ``'
  /Users/vamsee/Development/ruby/gems/fastlane-2.62.0/fastlane_core/lib/fastlane_core/analytics/analytics_session.rb:194:in `operating_system_version'
  /Users/vamsee/Development/ruby/gems/fastlane-2.62.0/fastlane_core/lib/fastlane_core/analytics/analytics_session.rb:71:in `action_launched'
  /Users/vamsee/Development/ruby/gems/fastlane-2.62.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:64:in `run!'
  /Users/vamsee/Development/ruby/gems/commander-fastlane-4.4.5/lib/commander/delegates.rb:15:in `run!'
  /Users/vamsee/Development/ruby/gems/fastlane-2.62.0/fastlane/lib/fastlane/commands_generator.rb:303:in `run'
  /Users/vamsee/Development/ruby/gems/fastlane-2.62.0/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
  /Users/vamsee/Development/ruby/gems/fastlane-2.62.0/fastlane/lib/fastlane/cli_tools_distributor.rb:66:in `take_off'
  /Users/vamsee/Development/ruby/gems/fastlane-2.62.0/bin/fastlane:20:in `<top (required)>'
  /Users/vamsee/Development/ruby/bin/fastlane:23:in `load'
  /Users/vamsee/Development/ruby/bin/fastlane:23:in `<top (required)>'

DEBUG [2017-10-19 00:20:24.13]: All plugins are up to date


```

</details>

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.13                                       |
| Ruby                        | 2.4.2                                       |
| Bundler?                    | true                                        |
| Git                         | git version 2.13.5 (Apple Git-94)           |
| Installation Source         | ~/Development/ruby/bin/fastlane             |
| Host                        | Mac OS X 10.13 (17A405)                     |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/2.4.2/lib            |
| OpenSSL Version             | OpenSSL 1.0.2l  25 May 2017                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 9.0.1                                       |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# Customise this file, documentation can be found here:
# https://github.com/fastlane/fastlane/tree/master/docs
# All available actions: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "2.58.0"

$:.unshift File.dirname(__FILE__)
require 'Onkore.rb'

default_platform :ios

platform :ios do
  before_all do
      
    ENV["FASTLANE_XCODE_LIST_TIMEOUT"] = "120"
    Onkore.setupEnv
    
    if ENV['FL_CHECKOUT']
        puts "Checking out code to ensure build directory is clean ..."
        sh("git checkout ../")
    end
    ensure_git_status_clean unless ENV['FL_NO_ENSURE_CLEAN']
    ENV["FASTFILE_DIR"] =  File.expand_path(File.dirname(File.dirname(__FILE__)))
  end


  desc "Parameterized Jenkins lane. If params are omitted will default to Bachelor"
  lane :jenkins do |options|
    cocoapods unless ENV['FL_NO_PODS']
    type = 'development'
    app = Onkore.getApp(options,type)
    match(app_identifier:app['appid'] , readonly: true, type: type)
    suffixRec = Onkore.getSuffix(type)
    profile= suffixRec['profile'] + app['appid']
    team=ENV['FASTLANE_TEAM_ID']

    gym(
        workspace: 'Onkore.xcworkspace',
        scheme: app['scheme'],
        export_method: "development",
        output_directory: "archives",
      com.onkore.BachelorFantasyUSA.Debug' DEVELOPMENT_TEAM=XXXXXX",
        xcargs: "PROVISIONING_PROFILE=profile DEVELOPMENT_TEAM=#{team}",
        clean: true, include_bitcode:false
    )
  end




  after_all do |lane|
    # This block is called, only if the executed lane was successful
  end

  error do |lane, exception|
     #slack(
     #  message: exception.message,
     #  success: false
     #)
  end
end


# More information about multiple platforms in fastlane: https://github.com/fastlane/fastlane/blob/master/docs/Platforms.md
# All available actions: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md

# fastlane reports which actions are used
# No personal data is recorded. Learn more at https://github.com/fastlane/enhancer

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.62.0  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                                      | Version | Update-Status |
| ------------------------------------------- | ------- | ------------- |
| fastlane-plugin-upload_symbols_to_new_relic | 0.1.1   | ✅ Up-To-Date  |
| fastlane-plugin-appicon                     | 0.10.0  | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                                         | Version      |
| ------------------------------------------- | ------------ |
| did_you_mean                                | 1.1.0        |
| bundler                                     | 1.15.4       |
| io-console                                  | 0.4.6        |
| CFPropertyList                              | 2.3.5        |
| concurrent-ruby                             | 1.0.5        |
| i18n                                        | 0.9.0        |
| minitest                                    | 5.10.3       |
| thread_safe                                 | 0.3.6        |
| tzinfo                                      | 1.2.3        |
| activesupport                               | 4.2.10       |
| public_suffix                               | 2.0.5        |
| addressable                                 | 2.5.2        |
| babosa                                      | 1.0.2        |
| claide                                      | 1.0.2        |
| fuzzy_match                                 | 2.0.4        |
| nap                                         | 1.1.0        |
| cocoapods-core                              | 1.3.1        |
| cocoapods-deintegrate                       | 1.0.1        |
| cocoapods-downloader                        | 1.1.3        |
| cocoapods-plugins                           | 1.0.0        |
| cocoapods-search                            | 1.0.0        |
| cocoapods-stats                             | 1.0.0        |
| netrc                                       | 0.11.0       |
| cocoapods-trunk                             | 1.3.0        |
| cocoapods-try                               | 1.1.0        |
| colored2                                    | 3.1.2        |
| escape                                      | 0.0.4        |
| fourflusher                                 | 2.0.1        |
| gh_inspector                                | 1.0.3        |
| molinillo                                   | 0.5.7        |
| ruby-macho                                  | 1.1.0        |
| nanaimo                                     | 0.2.3        |
| xcodeproj                                   | 1.5.2        |
| cocoapods                                   | 1.3.1        |
| colored                                     | 1.2          |
| highline                                    | 1.7.8        |
| commander-fastlane                          | 4.4.5        |
| declarative                                 | 0.0.10       |
| declarative-option                          | 0.1.0        |
| unf_ext                                     | 0.0.7.4      |
| unf                                         | 0.1.4        |
| domain_name                                 | 0.5.20170404 |
| dotenv                                      | 2.2.1        |
| excon                                       | 0.59.0       |
| multipart-post                              | 2.0.0        |
| faraday                                     | 0.13.1       |
| http-cookie                                 | 1.0.3        |
| faraday-cookie_jar                          | 0.0.6        |
| faraday_middleware                          | 0.12.2       |
| fastimage                                   | 2.1.0        |
| jwt                                         | 2.1.0        |
| little-plugger                              | 1.1.4        |
| multi_json                                  | 1.12.2       |
| logging                                     | 2.2.2        |
| memoist                                     | 0.16.0       |
| os                                          | 0.9.6        |
| signet                                      | 0.8.1        |
| googleauth                                  | 0.6.1        |
| httpclient                                  | 2.8.3        |
| mime-types-data                             | 3.2016.0521  |
| mime-types                                  | 3.1          |
| uber                                        | 0.1.0        |
| representable                               | 3.0.4        |
| retriable                                   | 3.1.1        |
| google-api-client                           | 0.13.6       |
| json                                        | 2.1.0        |
| mini_magick                                 | 4.5.1        |
| multi_xml                                   | 0.6.0        |
| plist                                       | 3.3.0        |
| rubyzip                                     | 1.2.1        |
| security                                    | 0.1.3        |
| slack-notifier                              | 1.5.1        |
| terminal-notifier                           | 1.8.0        |
| unicode-display_width                       | 1.3.0        |
| terminal-table                              | 1.8.0        |
| tty-screen                                  | 0.5.0        |
| word_wrap                                   | 1.0.0        |
| rouge                                       | 2.0.7        |
| xcpretty                                    | 0.2.8        |
| xcpretty-travis-formatter                   | 0.0.4        |
| fastlane-plugin-appicon                     | 0.10.0       |
| fastlane-plugin-upload_symbols_to_new_relic | 0.1.1        |
| xcode-install                               | 2.3.1        |
</details>

*generated on:* **2017-10-19**
</details>

**Comment by igobrilhante** [comment]
I've got the same issue using version 2.62.0. Any news?

**Comment by tokinonagare** [comment]
Same issue using 2.62.0. Try reinstall, but not help : (

**Comment by lakamsani** [comment]
Yes, definitely 2.62.0. My workaround is to go back to 2.58.0 which was the last working version for me, haven't tested with 2.60 etc. Do this if you are using `bundler`. 

1.  `gem uninstall fastlane`
2.  Change `fastlane` gem line to look like this: `gem 'fastlane', '2.58.0'`
3.  `bundle install`
4. `bundle exec fastlane <your lane>`

**Comment by mpirri** [comment]
This is a known issue with `fastlane` 2.62.0 and case-sensitive file systems. We will ship a fix for it shortly - the fix is in PR #10630.  Thanks and sorry for the inconvenience! 🚀
